jobbergate-api/jobbergate_api/tests/apps/job_submissions/test_routers.py

```diff
@@ -108,6 +108,90 @@ async def test_create_job_submission__with_client_id_in_token(
     assert job_submission.execution_parameters.comment == "I am a comment"
 
 
+@pytest.mark.asyncio
+async def test_create_job_submission__without_execution_parameters(
+    fill_application_data,
+    fill_job_script_data,
+    fill_job_submission_data,
+    client,
+    inject_security_header,
+    time_frame,
+):
+    """
+    Test POST /job-submissions/ correctly creates a job_submission.
+
+    This test proves that a job_submission is successfully created via a POST request to the /job-submissions/
+    endpoint. We show this by asserting that the job_submission is created in the database after the post
+    request is made, the correct status code (201) is returned. We also show that the ``client_id``
+    is pulled from the token and the created job_submission is connected to that client id.
+
+    This test is the same as the previous one, but it does not include the ``execution_parameters``
+    in the payload, since they are optional.
+    """
+    inserted_application_id = await database.execute(
+        query=applications_table.insert(),
+        values=fill_application_data(),
+    )
+    inserted_job_script_id = await database.execute(
+        query=job_scripts_table.insert(),
+        values=fill_job_script_data(application_id=inserted_application_id),
+    )
+
+    inject_security_header(
+        "owner1@org.com",
+        Permissions.JOB_SUBMISSIONS_EDIT,
+        client_id="dummy-cluster-client",
+    )
+    create_data = fill_job_submission_data(
+        job_script_id=inserted_job_script_id,
+        job_submission_name="sub1",
+        job_submission_owner_email="owner1@org.com",
+    )
+
+    # Removed defaults to make sure these are correctly set by other mechanisms
+    create_data.pop("status", None)
+    create_data.pop("client_id", None)
+
+    with mock.patch(
+        "jobbergate_api.apps.job_submissions.routers.get_job_properties_from_job_script"
+    ) as mocked:
+        mocked.return_value = JobProperties()
+        with time_frame() as window:
+            response = await client.post("/jobbergate/job-submissions/", json=create_data)
```

Comment:
part of me feels weird to keep this forward slash. I can't remember exactly what happened but I remember me and @dusktreader being stuck for a long in Mexico trying to find out a problem and at the end we discovered the problem was a damn forward slash. Do you mind removing it and testing as our frontend and the jobbergate cli uses

Reply:
The test passes without the forward slash `response = await client.post("/jobbergate/job-submissions", json=create_data)`.

Almost all tests on the routes use it, so the investigation and removal of them may be out of scope for this PR. I agree that is a valid point to keep in mind.